src/Apps/Artist/Routes/CV/CVItem.tsx

```diff
@@ -89,6 +89,12 @@ class CVItem extends Component<CVItemProps, CVItemState> {
     )
 
     return (
+      /**
+       * TODO:  replace with new responsive component
+       *
+       * Sometimes conditional rendering utilities are helpful, though the below
+       * would most likely benefit from a refactor since the code is a bit crazy
+       */
```

Comment:
I think this isn't as bad as it seems. Can I have a go at updating it?

Reply:
Not sure if you’re asking @damassi or me, but I’m more than happy for you to have a go 👍

Reply:
heh, I think its fine for now, was just a bit hard to reason about in terms of the new API 😄 We'll have to return to all of these conditional items in time.